content/blog/azure-native-defaultazurecredential-and-private-cloud/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Azure Native 3.8: Unified Credentials and Private Clouds"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2025-09-02T15:42:22-07:00
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: |
+    Pulumi Azure Native v3.8 delivers new credential modes (based on DefaultAzureCredential)
+    and private cloud support (based on ARM_METADATA_HOSTNAME).
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - eron-wright
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - azure-native
+    - azure
+    - features
+    - releases
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
+
+# Here are some examples of posts we have made in the past for inspiration:
+# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
+# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
+# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
+# https://twitter.com/PulumiCorp/status/1763265391042654623
+# https://twitter.com/PulumiCorp/status/1762900472489185492
+# https://twitter.com/PulumiCorp/status/1755637618631405655
+
+social:
+    twitter:
+    linkedin:
+
+---
+
+Today we're excited to announce Azure Native Provider v3.8, featuring several enhancements that simplify authentication
+and extend support to private Azure environments. These updates make it easier than ever to manage Azure infrastructure across diverse deployment scenarios.
```

Comment:
What do we mean by "diverse" here? "Varied"? Or "multiple"? Or "multi-tenant" in the sense that there's one central management?